### PR TITLE
[tests] resolves skipped HF tests: 3rd batch

### PR DESCRIPTION
**Pull request by pavel-esir**
### Details:
 - resolved skipped HF tests: 3rd batch

### Tickets:
 - [CVS-124981](https://jira.devtools.intel.com/browse/CVS-124981)

### CODE REVIEW

File: tests/model_hub_tests/torch_tests/test_hf_transformers.py [hunk 1]
```diff
@@ -125,6 +125,39 @@ def load_model(self, name, type):
             preprocessor = CLIPFeatureExtractor.from_pretrained(name)
             encoded_input = preprocessor(self.image, return_tensors='pt')
             example = dict(encoded_input)
+        elif 'xclip' in mi.tags:
+            from transformers import XCLIPVisionModel
+            
+            model = XCLIPVisionModel.from_pretrained(name)
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/test_hf_transformers.py:131]: Please add `**model_kwargs` for each model

File: tests/model_hub_tests/torch_tests/test_hf_transformers.py [hunk 2]
```diff
@@ -125,6 +125,39 @@ def load_model(self, name, type):
...
+            example = (torch.randn(*(16, 3, 224, 224), dtype=torch.float32), )  # needs video as input
+        elif 'audio-spectrogram-transformer' in mi.tags:
+            example = (torch.randn(*(1, 1024, 128), dtype=torch.float32), )
+        elif 'mega' in mi.tags:
+            from transformers import AutoModel
+            
+            model = AutoModel.from_pretrained(name)
+            example = model.dummy_inputs
+            example.update({'output_attentions': True, 'output_hidden_states': True, 'return_dict': True})
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/test_hf_transformers.py:140]: Is this really required? will it work without this?

Comment (pavel-esir) [reply tests/model_hub_tests/torch_tests/test_hf_transformers.py:140]: without this flags, there are `None`s on outputs and model becomes untraceable
image

Comment (eaidova) [reply tests/model_hub_tests/torch_tests/test_hf_transformers.py:140]: is it possible to use  model config for manipulate with this values?
e.g. model.config.return_dict = True

Comment (pavel-esir) [reply tests/model_hub_tests/torch_tests/test_hf_transformers.py]: done: replaced this with setting config

**Review by mvafin — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**